tests/src/io.jsonwebtoken/jjwt-impl/0.11.5/src/test/java/io_jsonwebtoken/jjwt_impl/Jjwt_implTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package io_jsonwebtoken.jjwt_impl;
+
+import io.jsonwebtoken.CompressionCodecs;
+import io.jsonwebtoken.JwtParserBuilder;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.io.Encoders;
+import io.jsonwebtoken.security.Keys;
+import org.junit.jupiter.api.Test;
+
+import javax.crypto.SecretKey;
+import java.security.KeyPair;
+import java.util.Date;
+import java.util.UUID;
+
+class Jjwt_implTest {
```

Comment:
Should I say that after executing `./gradlew scaffold --coordinates io.jsonwebtoken:jjwt-impl:0.11.5`, is it a strange thing to use underscore nomenclature to generate class names? Do i need to fix it?

Reply:
- I'm not quite sure what the final format should be. I expect that after the modification for the scaffold task is in place, I will modify the class name.